.flake8

```diff
@@ -1,6 +1,8 @@
 [flake8]
 max-line-length = 99
-select: E,W,F,C,N
+max-complexity = 10
+select = E,W,F,C,N
```

Comment:
This selection will not warn for docstrings and other things. Check out same file in alertmanager/karma.

Reply:
Thanks. Will fix in a future commit.